EEP_Gleisplan.html

```diff
@@ -1232,8 +1247,17 @@
 	document.getElementById('css-fontSize-variables').textContent =
 		`svg {
 			--normal-font-size: ${4 * factor}px;
-		}`	
-}</script>
+		}`;
+}
+
+// Listen for checkbox change events
+document.addEventListener('DOMContentLoaded', function() {
+	document.querySelector('.fieldset').addEventListener('change', function(ev) {
```

Comment:
Currently listening for all change events inside the `fieldset` (which currently are just checkboxes).
Maybe make this more specifc to checkboxes.